infra/docker-compose/dev/istio/minimal-operator.yaml

```diff
@@ -5,38 +5,22 @@ spec:
     global:
       proxy:
         autoInject: disabled
-      useMCP: false
-      # The third-party-jwt is not enabled on all k8s.
-      # See: https://istio.io/docs/ops/best-practices/security/#configure-third-party-service-account-tokens
-      jwtPolicy: first-party-jwt
-    # Patch to fix validation error: port http2/80 in gateway cluster-local-gateway 
-    # invalid: targetPort is set to 0, which requires root 
     gateways:
       istio-ingressgateway:
         runAsRoot: true
-
-  addonComponents:
-    pilot:
-      enabled: true
-    prometheus:
-      enabled: false
-
   components:
     ingressGateways:
     - name: istio-ingressgateway
       enabled: true
-    - name: cluster-local-gateway
-      enabled: true
-      label:
-        istio: cluster-local-gateway
-        app: cluster-local-gateway
       k8s:
         service:
-          type: ClusterIP
+          type: LoadBalancer
```

Comment:
whats the motivation for the change from `ClusterIp` to `LoadBalancer`?

Reply:
Good question. AFAIK, Ingress gateways are typically set to `LoadBalancer` to be accessible outside the cluster. I think `ClusterIP` can also be used with k3d in which case the Istio ingress gateway needs to be exposed differently (see `.github/actions/run-cluster-init/action.yaml`).